plugins/services/src/js/containers/tasks/TasksView.js

```diff
@@ -1,44 +1,39 @@
 import classNames from "classnames";
 import mixin from "reactjs-mixin";
 import { Tooltip } from "reactjs-components";
-import React from "react";
+import React, { PropTypes } from "react";
+import { routerShape } from "react-router";
 
 import DCOSStore from "#SRC/js/stores/DCOSStore";
+import DSLFilterField from "#SRC/js/components/DSLFilterField";
+
 import FilterBar from "#SRC/js/components/FilterBar";
-import FilterButtons from "#SRC/js/components/FilterButtons";
 import FilterHeadline from "#SRC/js/components/FilterHeadline";
-import FilterInputText from "#SRC/js/components/FilterInputText";
 import Icon from "#SRC/js/components/Icon";
 import SaveStateMixin from "#SRC/js/mixins/SaveStateMixin";
 import StringUtil from "#SRC/js/utils/StringUtil";
 import { isSDKService } from "#SRC/js/utils/ServiceUtil";
 
+import TaskStatusDSLSection from "../../components/dsl/TaskStatusDSLSection";
+import TaskZoneDSLSection from "../../components/dsl/TaskZoneDSLSection";
+import TaskRegionDSLSection from "../../components/dsl/TaskRegionDSLSection";
+import FuzzyTextDSLSection from "../../components/dsl/FuzzyTextDSLSection";
+
 import ServiceStatusTypes from "../../constants/ServiceStatusTypes";
 import GraphQLTaskUtil from "../../utils/GraphQLTaskUtil";
```

Comment:
kinda confusing having `GraphQLTaskUtil ` even though we don't use it. No need to change in this PR but would be nice a follow up PR renaming/refactoring (I can do that too if you want to)

Reply:
@weblancaster would love if you could fix the name.